### DISCUSSION TITLE
Download ignore empty folders

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#226] [General]
I noticed that when we download folder we do not get an exact copy of it because the download process ignore any empty folders inside the folder you download.
For example if you have on the server folder “1” and inside it there are two folders : folder “11” ( not empty, it has some files inside it) and folder “12” ( empty folder or contains another empty folders with no files inside).
Now, When we download folder “1” , we get folder “1” and folder “11” inside it ONLY, which is not exact copy of the folder “1” on the server.
I suggest not ignoring the empty folders during the downloading.
Please advise

**Comment by rejetto** [comment]
Is this something that's just better or do you need it?

**Comment by GeorgeGeorgeGe** [comment]
For me, yes i need it.
i Think without this feature, user may thing that the download is not done successful, he may doubt that other non-empty folders may not be downloaded as well because he is not seeing an exact copy of the original.
Thanks

**Comment by rejetto** [comment]
Now zip will include empty folders.
Find this in latest beta #241 or next release.
